src/WebApp/Pages/Shared/_AttachmentLinkPartial.cshtml

```diff
@@ -16,7 +16,7 @@
 else
 {
     <a asp-page="Attachment" asp-route-id="@Model.Id.ToString()" asp-route-fileName="@Model.FileName" target="_blank"
-       class="text-decoration-hover rounded d-block p-2 no-external-link">
+       data-fancybox="gallery" class="text-decoration-hover rounded d-block p-2 no-external-link">
```

Comment:
Remove the change on this line. I don't want the gallery to open for non-image files.